<?xml version="1.0"?>
<launch>
    <node name="rviz_udpServer" pkg="motion_udp" type="rviz_udpServer" >
        <param name="local_port" value="54321"/>
<!--        <param name="remote_port" value="8080"/>
        <param name="remote_ip" value="192.168.0.1"/>
-->
        <param name="packet_size" value="208"/>
    </node>

<!--  <arg name="model" />
  <arg name="gui" default="true" />-->
  <!-- <param name="robot_description"  textfile="$(find robot_description)/robots/robot_description.urdf" /> -->
  <param name="robot_description"  textfile="$(find mini)/urdf/mini.urdf" />
<!--  <param name="use_gui" value="$(arg gui)" />-->
  <node  name="jointpublish"  pkg="motion_udp"  type="jointpublish" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find robot_description)/urdf.rviz" /> -->

    <!--<node name="rviz" pkg="rviz" type="rviz" args="-d $(find mini)/urdf.rviz" />-->

</launch>
